src/base-input.js

```diff
@@ -482,14 +493,29 @@ export default class BaseInput extends LitElement {
    * @returns {string} Error string.
    */
   getErrorMessage() {
-    if (this.error) {
-      return this.error;
-    }
+    let message = '';
+
     if (this.setCustomValidity) {
-      return this.setCustomValidity;
+      // return this.setCustomValidity;
+      message = this.setCustomValidity;
+    } else if (this.error) {
+      // return this.error;
+      message = this.error;
+    } else {
+      // return this.internalError;
+      message = this.internalError;
     }
 
-    return this.internalError;
+    this.dispatchEvent(new CustomEvent('auroInput__helpText', {
+      bubbles: true,
+      cancelable: false,
```

Comment:
This is redundant - [cancelable: false is the default](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event).